package.json

```diff
@@ -17,6 +17,7 @@
     "chart.js": "^2.9.3",
     "dotenv": "^8.2.0",
     "env-cmd": "^10.1.0",
+    "framer-motion": "^2.3.0",
```

Comment:
@AuraOfDivinity I am not happy with this npm package, as it has 8 dependencies, can't do it with css transitions??